scripts/tests/test_zkai_attention_kv_stwo_native_two_head_seq32_bounded_softmax_table_proof_input.py

```
@@ -0,0 +1,202 @@
+            root = pathlib.Path(tmp)
+            json_out = root / "json" / "out.json"
+            tsv_out = root / "tsv" / "out.tsv"
+            gate.write_outputs(payload, json_out, tsv_out)
+            self.assertTrue(json_out.is_file())
+            self.assertTrue(tsv_out.is_file())
+
+    def test_write_outputs_rejects_same_path(self):
+        payload = gate.build_payload()
+        with tempfile.TemporaryDirectory(dir=gate.EVIDENCE_DIR) as tmp:
+            out = pathlib.Path(tmp) / "same.out"
+            with self.assertRaisesRegex(
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)